policies/adding-new-maintainers.md

```diff
@@ -0,0 +1,23 @@
+This document describes the process for adding new people to maintainer roles.
+"Maintainer" here basically means any role with github write powers. The
+process is in the hands of the Coordination Committee according to
+[APE0](https://github.com/astropy/astropy-APEs/blob/main/APE0.rst), which also
+requires the comment period. The process is:
+
+1. Either the Coordination Committee or any community member including the 
```

Comment:
Would it better here to elaborate what you mean by "any community member"? I assume it is anyone broadly working with Astropy, but I am not exactly sure.

Reply:
Yes, but in the latest commit I ended up removing this section.  But we should make a separate document about the nomination process to answer this question better, @aaryapatil !